server/lib/nl/common/utterance.py

```diff
@@ -174,4 +174,8 @@ class Utterance:
   # Past complete context for insight flow.
   insight_ctx: Dict = field(default_factory=dict)
   # Test param.
-  test: str = ''
\ No newline at end of file
+  test: str = ''
+  # i18n param.
```

Comment:
Wonder if we should have just 1 field `i18n_lang`?

Reply:
Done.